modules/running-distributed-data-science-workloads-from-ds-pipeline.adoc

```diff
@@ -0,0 +1,133 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-from-ds-pipeline_{context}"]
+= Running distributed data science workloads from data science pipelines
+
+[role='_abstract']
+To run a distributed data science workload from a data science pipeline, you must first update the pipeline to include a link to your Ray cluster image.
+
+.Prerequisites
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
```

Comment:
They need to be logged in to the dashboard to do most of the tasks in the procedure.